PW-16192 Up to Top button polish

zacharymorel opened the pull request:
## Description
PR to fix several small css inconsistencies between the mock up and what was built. 

## Screenshots
<img width="420" alt="Screen Shot 2020-11-17 at 10 08 13 AM" src="https://user-images.githubusercontent.com/26075258/99407270-e5d77f00-28bc-11eb-9ef5-fbfe16ec5ee0.png">
<img width="487" alt="Screen Shot 2020-11-17 at 10 08 02 AM" src="https://user-images.githubusercontent.com/26075258/99407283-ea039c80-28bc-11eb-8b8c-152cdba2785c.png">
<img width="1429" alt="Screen Shot 2020-11-17 at 10 08 22 AM" src="https://user-images.githubusercontent.com/26075258/99407289-ebcd6000-28bc-11eb-95fb-3ceabe55b20f.png">
<img width="1433" alt="Screen Shot 2020-11-17 at 10 08 35 AM" src="https://user-images.githubusercontent.com/26075258/99407297-ee2fba00-28bc-11eb-9bc6-2d33dba5e68c.png">


## Acceptance criteria
- [x] Ensure code is accessible. 
- [x] Center the Button between the footer and the "Last Updated" text when it is resting on the bottom of the footer.
- [x] Add desired hover state specified in user story.
- [x] Tweak mobile to what is specified in the User Story.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

Commit:
Polishing CSS

Commit:
Polishing CSS

ncksllvn commented on src/applications/static-pages/sass/static-pages.scss:
```diff
@@ -127,5 +134,20 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
+      padding: 0;
+      width: 44px;
+    }
   }
```
Could we move the top button's style into its own module file so that it has its own jurisdiction? Somewhere in `static-pages/sass/modules`

I'm also wondering if can use the mobile-first pattern here instead of `media-maxwidth`? Would mobile-first be a big hassle?

zacharymorel replied:
I was wondering about extracting the code out of the _entry_ file and i'm glad you said something. 

As far as the mobile first, we can do that. It's just rewriting the same properties that is on the `.usa-button` class to the custom sass styling here. I went with Max Width media queries because it saved a little time and code duplication.

ncksllvn replied:
If t's a big pain and leads to dup code then this is probably fine. We have just seen weird UI bugs sometimes from when certain elements have breakpoints at width "x" and others at "x + 1px" and it's hard to tell when that could happen in media queries.

zacharymorel replied:
Yeah, it's only a few extra properties really. So not a big deal. Sticking with the repo standard is probably best. Thanks man.

mpelzsherman committed:
Fix node-health_care_local_health_service transformer (#14983)

An error in this transformer was causing the entire health services section to be empty
in all VAMC location pages.

Mottie committed:
[Development] HLR: Display contact info from Vets Profile (#14977)

* Show profile data on review page

* Fix form data updates

* Update data & transformers

* Show profile data on contact page

* Re-organize files

* Update unit tests

* Update formatAddress unit tests

* Fix capitalization

* Process address directly

* Hide components on review & submit page

* Update unit tests

zacharymorel committed:
Polished after comments

zacharymorel committed:
Polished after comments

micahchiang committed:
12083 migrate 686 e2e to cypress (#14957)

* initial cypress files

* re-order and fix a11y issues

* stub in progress routes

* spouse-divorce test files

* Final e2e cypress files

* Consolidate tests

* a11y fixes

* skip unrelated test

* split into separate specs

* change order

* modify test data

* a final hope

U-DON committed:
Mocha v8.2.1 (#13982)

* Upgrade to Mocha 8.1.1.

* Attempt to convert configuration to Mocha 8.

* Some setup fixes.

* Increased timeout.

* Testing Mocha 8 parallel tests in Jenkins.

* Disabled parallel tests by default.

* Run full coverage in both Jenkins and Circle.

* Fixed NPM script argument syntax.

* Corrected script path in Circle job.

* Removed comment to make Mocha config a valid JSON file.

* Added Choma to Mocha config requires.

* Minor version bump.

* Reordered requires to be the same order as the opts config.

* Reverted ordering of requires.

* Disabled problematic tests.

* Fixed breaking Pre-need tests.

* Removed references to mocha.opts.

* Removed unused helper.

* Removed unused script.

ndsprinkle committed:
Verify service period is defined before checking for branch and test fix for ConfirmationPoll (#14979)

jhonny218 committed:
Add encode to fieldDocument (#14989)

shefalinayak-tw committed:
Contact Us - updates Cypress test data (#14825)

erikphansen committed:
Split up HCA reducers (#14984)

So that the Dashboard can pull in only the part that it needs.

timwright12 committed:
Feature/upgrade testing tools (#14991)

* upgrade testing tools

narin committed:
Add GA event to measure eligibility loading time (#14969)

cohnjesse committed:
Adjusted copy (#14994)

Co-authored-by: Jesse Cohn <jesse.cohn@teamadhoc.us>

cohnjesse committed:
Updated events per new spec (#14956)

* Updated events per new spec

* added cta click event

* Moved session storage call

Co-authored-by: Jesse Cohn <jesse.cohn@teamadhoc.us>

cohnjesse committed:
15484 ui updates (#14944)

* Adjusted breadcrumb name

* Added line height and changed intro content

* Adjusted margin below paragraph

* Adjusted paragraphs

* adjusted mobile and desktop padding

* Adjusted link text

* Sentence cased labels

* adjusted alignment and bumped version number

* Corrected aligned columns

* added right aligned

Co-authored-by: Jesse Cohn <jesse.cohn@teamadhoc.us>

mpelzsherman committed:
Add missing fieldMedia data for health_care_region_page (#14995)

brangi committed:
FL - Search this area (#14831)

* Start search area implementation

* Add useEffect to update search area

* clean up code

* Fixes, set accurate bounds

* update search string and context and cleanup

* fix to calculated bounds

* Create search area classes and setup container for adding styles

* add search area dims for small screens

* fix update url params when search area and render markers if results in the store

* remove unnecessary listener

* Setup constants for requests values

* Fix recordPan events for new map

* css clean up

* minor

* Add ga event for search area

Co-authored-by: Gibran Rodriguez <gribranrodriguez@USRMonitorsMBP2.attlocal.net>

bradl3yC committed:
Add the even to a DL analytics event (#15002)

narin committed:
Remove use of vaParent for v2 facility page and throughout flow where unneeded (#14936)

* Redirect users registered to Sacramento to v1 facility page

* Rename selector

* One more selectUseFlatFacilityPage test for good measure

* Start removing vaParent from v2 flow

* Add 612 mocks

* Replace uses of vaParent

* Add 612 mocks

* Revert mocks index.  Add confirmed va 612 mocks

* Add confirmed va 612 mocks

* Fix spec.  Fix old facility page selection

* Fix tests

* Fix DateTimeSelectPage tests

* Add feature toggle for sacramento flat page for testing

* Rename feature toggle vars

* Fix feature toggle selector

* Removed unused selector

* Add flag to featureFlagNames

* Simplify sacramento selector logic

* Fix WaitTimeAlert test

ncksllvn reviewed (approved):
LGTM

jhonny218 reviewed (approved):
LGTM

LeKeve committed:
Connect app directory to apps api (#14985)

* Remove pagination component and add CTA buttons in results list

* Remove FAQ text

* Add fetch scopes actions and cleanup mock data for testing

* Add api methods for scopes

* Update app endpoint and fix logo overflow

* Update scopes logic and fix app result params

* Remove unused components and fix tests

* Fix aria attribute and margin issues

* Fix widget types for app directory

* Remove unused state and options param

* Fix typo in styles

* Update platform copy and fix tests

Mottie committed:
[Development] HLR: Filter out ineligible contestable issues (#14998)

* Filter out ineligible contestable issues

* Update tests & mock data

zacharymorel committed:
Merge branch 'pw-16192-button-bugs' of github.com:department-of-veterans-affairs/vets-website into pw-16192-button-bugs

zacharymorel commented:
Whelp, I did a rebase for this repo locally and I did not even consider it was going to trigger all those teams for a review.
Seems like github is handling git rebase weirdly?

Going to open a new a PR and ping @jhonnyoddball and @ncksllvn to get re-reviews on the two commits prior to the rebase.
